skl2onnx/_parse.py

```diff
@@ -6,186 +6,50 @@
 import warnings
 import numpy as np
 
-from .common._container import SklearnModelContainer
-from .common._topology import Topology, Variable, Operator, Scope, convert_topology
-from .common._registration import register_converter, register_shape_calculator, get_shape_calculator
-from .common.data_types import DataType, Int64Type, FloatType, StringType, TensorType, find_type_conversion
-from .common.data_types import FloatTensorType, Int64TensorType, SequenceType, DictionaryType
-from .common.utils import get_column_indices
-from .shape_calculators.Concat import calculate_sklearn_concat
-
-# Pipeline
 from sklearn import pipeline
-from sklearn.base import ClassifierMixin, RegressorMixin, ClusterMixin
-
-# Calibrated classifier CV
-from sklearn.calibration import CalibratedClassifierCV
-
-# Column Transformer
 from sklearn.compose import ColumnTransformer
-
-# Linear classifiers
-from sklearn.linear_model import LogisticRegression
-from sklearn.linear_model import SGDClassifier
-from sklearn.svm import LinearSVC
-
-# Linear regressors
-from sklearn.linear_model import ElasticNet
-from sklearn.linear_model import Lasso
-from sklearn.linear_model import LassoLars
-from sklearn.linear_model import LinearRegression
-from sklearn.linear_model import Ridge
-from sklearn.linear_model import SGDRegressor
-from sklearn.svm import LinearSVR
-
-# Multi-class
-from sklearn.multiclass import OneVsRestClassifier
-
-# Tree-based models
-from sklearn.ensemble import GradientBoostingClassifier
-from sklearn.ensemble import GradientBoostingRegressor
-from sklearn.ensemble import RandomForestClassifier
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.ensemble import ExtraTreesClassifier
-from sklearn.ensemble import ExtraTreesRegressor
-from sklearn.tree import DecisionTreeClassifier
-from sklearn.tree import DecisionTreeRegressor
-
-# Support vector machines
-from sklearn.svm import SVC, SVR, NuSVC, NuSVR
-
-# K-nearest neighbors
-from sklearn.neighbors import KNeighborsClassifier
-from sklearn.neighbors import KNeighborsRegressor
+from sklearn.base import ClassifierMixin, RegressorMixin, ClusterMixin
+from sklearn.svm import LinearSVC, SVC, NuSVC
 from sklearn.neighbors import NearestNeighbors
```

Comment:
Why have you left this import when it's not used in this file?

Reply:
It is used to select the appropriate parser.